spec.bs

```diff
@@ -594,13 +648,15 @@ To <dfn>create a request</dfn> given a [=URL=] |url|, and a [=string=] <dfn for=
     ::  "`no-store`"
     :   [=request/redirect mode=]
     :: "`error`"
+    :   [=request/mode=]
+    :: If |corsMode| is true, then "`cors`", else "`same-origin`".

```

Comment:
So this is complicated.

Same-origin is invalid here since the request's origin is an opaque origin. This step will blow up: https://fetch.spec.whatwg.org/#ref-for-concept-request-mode%E2%91%A8.

CORS also requires a valid origin for the response to be readable/not-a-network-error, unless of course the server response with `Access-Control-Allow-Origin: 'null'`. So what is the implementation doing here? Are we really fetching with an opaque initiator origin, _and_ conditionally with CORS mode?

Reply:
Here is the [comment](https://crsrc.org/c/content/browser/interest_group/interest_group_permissions_checker.cc;drc=adac219925ef5d9c9a954d189c2e4b8852a4bbed;l=123) in code explaining the cors and there was a todo saying "figure out if we really need cors". :)

Reply:
At least for this specific instance it's not, since the implementation does not use an opaque origin for this fetch.

Reply:
For the moment, I'll leave a TODO in here.